src/arc.ts

```diff
@@ -540,6 +644,9 @@ export function pollForAccountChanges(currentAccountAddress: Address | null, int
         try {
           getCurrentAccountFromProvider()
             .then(async (account: Address | null): Promise<void> => {
+              if (prevAccount && account && prevAccount !== account) {
+                await biconomy.logout();
```

Comment:
Note this logout have not the right result with the changes I'm asking you to make.  So needs to be changed somehow.

Reply:
Done